test/specs/commands/createTransactionRegisterSecondPassphrase.js

```diff
@@ -0,0 +1,133 @@
+								beforeEach(given.thePassphraseIsProvidedViaStdIn);
+								describe('Given the second passphrase is provided via the prompt', () => {
+									beforeEach(given.theSecondPassphraseIsProvidedViaThePrompt);
+									describe('When the action is called with the options', () => {
+										beforeEach(when.theActionIsCalledWithTheOptions);
+										it('Then it should get the passphrase from stdin', then.itShouldGetThePassphraseFromStdIn);
+										it('Then it should get the passphrase using the passphrase from stdin', then.itShouldGetThePassphraseUsingThePassphraseFromStdIn);
+										it('Then it should get the passphrase using the vorpal instance', then.itShouldGetThePassphraseUsingTheVorpalInstance);
+										it('Then it should get the passphrase with a repeated prompt', then.itShouldGetThePassphraseWithARepeatedPrompt);
+										it('Then it should get the second passphrase with a repeated prompt', then.itShouldGetTheSecondPassphraseWithARepeatedPrompt);
+										it('Then it should create a register second passphrase transaction using the passphrase and the second passphrase', then.itShouldHaveCreatedARegisterSecondPassphraseTransactionUsingThePassphraseAndTheSecondPassphrase);
+										it('Then it should resolve to the created transaction', then.itShouldResolveToTheCreatedTransaction);
+									});
+								});
+							});
+						});
+						describe('Given an options object with passphrase and second passphrase set to "stdin"', () => {
+							beforeEach(given.anOptionsObjectWithPassphraseAndSecondPassphrasePassphraseSetToStdIn);
+							describe('Given the passphrase and the second passphrase is provided via stdin', () => {
+								beforeEach(given.thePassphraseAndTheSecondPassphraseIsProvidedViaStdIn);
+								describe('When the action is called with the options', () => {
+									beforeEach(when.theActionIsCalledWithTheOptions);
+									it('Then it should get the passphrase from stdin', then.itShouldGetThePassphraseFromStdIn);
+									it('Then it should get the second passphrase from stdin', then.itShouldGetTheSecondPassphraseFromStdIn);
+									it('Then it should get the passphrase using the vorpal instance', then.itShouldGetThePassphraseUsingTheVorpalInstance);
+									it('Then it should get the passphrase with a repeated prompt', then.itShouldGetThePassphraseWithARepeatedPrompt);
+									it('Then it should create a register second passphrase transaction using the passphrase and the second passphrase', then.itShouldHaveCreatedARegisterSecondPassphraseTransactionUsingThePassphraseAndTheSecondPassphrase);
+									it('Then it should resolve to the created transaction', then.itShouldResolveToTheCreatedTransaction);
+								});
+							});
+						});
+						describe('Given an options object with passphrase set to "stdin" and second passphrase set to "file:/path/to/my/password.txt"', () => {
+							beforeEach(given.anOptionsObjectWithPassphraseSetToStdInAndSecondPassphraseSetTo);
```

Comment:
`given.anOptionsObjectWithPassphraseSetToInAndSecondPassphraseSetTo` and extract string from title